obs-packaging/download_image.sh

```diff
@@ -8,6 +8,9 @@ set -o nounset
 set -o pipefail
 set -o errtrace
 
+script_dir=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
```

Comment:
Ah, now I have to point out this looks to be the same thing but using a different method than your first commit? Should we pick one method and stick to it?

Reply:
I'd the former, it has clearer intent.